src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Ball {
+    private Color color;
+    private int number;
+
+    public Ball(Color color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "number - "
+                + number
+                + "  "
+                + "color - "
+                + color;
+    }
```

Comment:
I would recommend to place variables names on the same line with variables themselves, as it would be more readable. Like:
`"number - " + number`

Reply:
I overdid the rule with + every line)I changed that